layouts/partials/job/header.html

```diff
@@ -0,0 +1,11 @@
+<div class="h-60 xl:h-80">
+  <div class="container mx-auto z-10 relative flex items-center h-full">
+    <div class="flex items-center">
+      <img class="job-icon-lg mr-16 filter drop-shadow-lg" src='{{ print "/theme-assets/jobs/" (lower .Title) ".svg" }}' />
+      <h1 class="text-6xl font-semibold filter drop-shadow-lg">{{ .Title }}</h1>
+    </div>
+  </div>
+  <div class="bg-gray-600 absolute top-0 left-0 w-full -z-1" style="height: 460px;">
+    <img class="object-cover w-full h-full" src='{{ print "/theme-assets/jobs/" (lower .Title) "_background.png" }}' />
```

Comment:
Out of curiosity, wouldn't this be better?

```html
<img class="object-cover w-full h-full" src="/theme-assets/jobs/{{ lower .Title }}_background.png" }}' />
```

Reply:
Absolutely makes sense! Fixed.